### PR TITLE
PMM-6518 Dashboards datasource name change

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/1162

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -164,6 +164,27 @@ def add_datasources(api_key):
...
+    else:
+        print ' * Modifing Metrics Data Source'
+        r = requests.get('%s/api/datasources/name/Prometheus' % (HOST,), headers=grafana_headers(api_key))
+        data = json.loads(r.content)
+        data['jsonData']['timeInterval']='1s'
+        data['jsonData']['httpMethod']='POST'
+        data['readOnly'] = False
+        r = requests.put('%s/api/datasources/%i' % (HOST, data['id']), data=json.dumps(data), headers=grafana_headers(api_key))
+        print r.status_code, r.content
+        if r.status_code != 200:
```
Comment (askomorokhov) [inline import-dashboards.py:184]: Here can be `httplib.OK` as well? 
https://github.com/percona/pmm-server/pull/234/files#diff-b473c5eafebc34251b1785f042d44e4eR172

File: import-dashboards.py [hunk 2]
```diff
@@ -164,6 +164,27 @@ def add_datasources(api_key):
     r = requests.get('%s/api/datasources' % (HOST,), headers=grafana_headers(api_key))
     print ' * Datasources: %r %r' % (r.status_code, r.content)
     ds = [x['name'] for x in json.loads(r.content)]
+    if 'Metrics' not in ds:   # https://jira.percona.com/browse/PMM-6518
```
Comment (askomorokhov) [inline import-dashboards.py:167]: This function (`add_datasources`) for me looks too long.
Maybe it is time to split it?
```python
def add_datasources(api_key):
    _add_meterics_datasources(api_key)
    _add_prometheus_datasources(api_key)
    # ....
    _add_alert_manager_datasources(api_key)
```

**Review by askomorokhov — CHANGES_REQUESTED** [review]

File: import-dashboards.py (hunk 1 above)
Comment (adivinho) [reply import-dashboards.py]: Done.

File: import-dashboards.py (hunk 2 above)
Comment (adivinho) [reply import-dashboards.py:164]: It has been splitted,

**Review by askomorokhov — APPROVED** [review]
Looks good to me 👍 

Please update FB.

File: import-dashboards.py [hunk 3]
```diff
@@ -160,10 +160,30 @@ def delete_api_key(db_key, upgrade):
...
 
 
-def add_datasources(api_key):
-    r = requests.get('%s/api/datasources' % (HOST,), headers=grafana_headers(api_key))
-    print ' * Datasources: %r %r' % (r.status_code, r.content)
-    ds = [x['name'] for x in json.loads(r.content)]
+def _add_metrics_datasource(api_key, ds):
+    if 'Metrics' not in ds:   # https://jira.percona.com/browse/PMM-6518
+        print ' * Adding Metrics Data Source'
+        data = json.dumps({'name': 'Metrics', 'type': 'prometheus', 'jsonData': {'keepCookies': [], 'timeInterval': '1s', 'httpMethod': 'POST'}, 'url': 'http://127.0.0.1:9090/prometheus/', 'access': 'proxy', 'isDefault': True})
```
Comment (AlekSi) [inline import-dashboards.py:166]: Can two data sources (Metrics and Prometheus below) be the default?

File: import-dashboards.py [hunk 4]
```diff
@@ -160,10 +160,30 @@ def delete_api_key(db_key, upgrade):
...
+        r = requests.post('%s/api/datasources' % HOST, data=data, headers=grafana_headers(api_key))
+        print r.status_code, r.content
+        if r.status_code != httplib.OK:
+            print ' * Cannot add Metrics Data Source'
+            sys.exit(-1)
```
Comment (AlekSi) [inline import-dashboards.py:171]: Let's avoid copy&pasting those line above and below

File: import-dashboards.py [hunk 5]
```diff
@@ -160,10 +160,30 @@ def delete_api_key(db_key, upgrade):
...
+    else:
+        print ' * Modifing Metrics Data Source'
+        r = requests.get('%s/api/datasources/name/Prometheus' % (HOST,), headers=grafana_headers(api_key))
```
Comment (AlekSi) [inline import-dashboards.py:174]: Pretty sure that should not be `Prometheus`

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: import-dashboards.py (hunk 3 above)
Comment (adivinho) [reply import-dashboards.py:166]: Thanks. I have missed it.

File: import-dashboards.py (hunk 5 above)
Comment (adivinho) [reply import-dashboards.py:174]: We are not switching to Victoria metrics yet.

Screenshot_20201012_131957

Comment (AlekSi) [reply import-dashboards.py:174]: I understand that. But in this code you get Prometheus configuration (see URL)

File: import-dashboards.py (hunk 4 above)
Comment (adivinho) [reply import-dashboards.py]: Code has been reduced to one block at the end of the function.

**Review by AlekSi — APPROVED** [review]

**Pull request merged**